tests/storage/storage_migration/utils.py

```
@@ -34,9 +39,40 @@ def verify_vms_boot_time_after_storage_migration(
         current_boot_time = get_vm_boot_time(vm=vm)
         if initial_boot_time[vm.name] != current_boot_time:
             rebooted_vms[vm.name] = {"initial": initial_boot_time[vm.name], "current": current_boot_time}
+            collect_reboot_diagnostic_events(vm=vm)
     assert not rebooted_vms, f"Boot time changed for VMs:\n {rebooted_vms}"
 
 
+def collect_reboot_diagnostic_events(vm: VirtualMachineForTests) -> None:
+    """Collect event log entries related to reboot/shutdown."""
+    base_dir = get_data_collector_dir()
+
+    if OS_FLAVOR_WINDOWS == vm.os_flavor:
```

Ruff (0.15.20)

[warning] Yoda condition detected

Rewrite as `vm.os_flavor == OS_FLAVOR_WINDOWS`

(SIM300)